cirrus/server/tests/test_experiment_recipes.py

```diff
@@ -1,87 +1,77 @@
-import unittest
 from unittest.mock import MagicMock, patch
 
+import pytest
 import requests
 
-from cirrus.experiment_recipes import RemoteSettings
-from cirrus.sdk import SDK
-from cirrus.settings import context
 
-
-class TestFetchRecipes(unittest.TestCase):
-    def setUp(self):
-        self.sdk = SDK(context)
-
-    def get_remote_settings(self):
-        return RemoteSettings(self.sdk)
-
-    def test_get_recipes_is_empty(self):
-        rs = self.get_remote_settings()
+class TestFetchRecipes:
+    def test_get_recipes_is_empty(self, remote_setting):
+        rs = remote_setting
         assert rs.get_recipes() == {"data": []}
 
-    def test_update_recipes(self):
-        rs = self.get_remote_settings()
+    def test_update_recipes(self, remote_setting):
+        rs = remote_setting
```

Comment:
Why is this alias necessary?

Reply:
not needed, my brain reading and writing a code making sure doing correctly :D